### DISCUSSION TITLE
Errors in User Wiki, Tutorial 2 Injection of oil and water through a 2D micromodel

### DISCUSSION THREAD

**Discussion by ZZZYunpeng** [GeoChemFoam/GeoChemFoam#17] [Q&A] [answered]
Hi,
I found that there are some errors in 'Tutorial 2: Injection of oil and water through a 2D micromodel' 
The first paragraph said '
This test case considers multiphase flow in a 2D micromodel. The image can be found in the $GCFOAM_IMG/raw directory with the name HM12_2_4. The domain is of size 1.8cm by 1.8 cm. The two fluids are water (density = 1000 kg/m^3, viscosity = 1 mPa.s) and oil (density = 750 kg/m^3, viscosity=1.5 mPa.s). The surface tension is set to 30 mN/m. _**The pore walls are set to be oil-wet with a contact angle of 45 degrees. The domain is initially filled with 25% water and 75% oil and we inject water from the left boundary**_ with a flow rate of 3.5e-8 m3/s, which corresponds to a capillary number of ~ 10-3.'

Maybe contact angle of 45 degrees is water-wet, and the domain is initially filled with 25% oil and 75% water and we inject oil from the left boundary. Because the example looks like this.

**Comment by julienmaes** [comment, marked as answer]
Hi Yunpeng,

Sorry for the confusion. In the paraview images, water is red and oil blue. We're showing alpha.water which with the default paraview colormap, is red for 1 and blue for 0. We'll be sure to invert the color scheme for next release.

Regards,

Julien

**Reply by ZZZYunpeng** [reply]
Thank you for your reply.
From the process of simulation, we can find that the bule (oil) is being injected from the left boundary and the wall is water-wet from the result.
So is my understanding right? 
In this example, contact angle of 45 degrees is water-wet, and the domain is initially filled with 25% oil and 75% water and we inject oil from the left boundary. Because the example looks like this.

Another question is about the setting of contact angle.
In OpenFOAM software, if the contact angle is set to 45 degrees, dose it means that it is water-wet?  Or is the wettability related to injection phase? For example, I set the Inlet-fixedvalue to 0 in the alpha.water file, which means that the injection phase is oil. And I set contact angle to 45 degrees, dose it means that it is oil-wet?

**Reply by julienmaes** [reply]
Ah yes, it's a typo, it's 25% oil and 75% water. The rest is correct. The walls are water-wet. Cheers
